### DISCUSSION TITLE
6.3.7 엔드포인트 테스팅 관련 질문드립니다.

### DISCUSSION THREAD

**Discussion by Daehyeon2yam** [fsoftwareengineer/todo-application#74] [질문과 답변] [answered]
안녕하세요 작가님 

로컬환경에서 접속에 문제 없음은 확인했습니다.
<img width="1078" alt="스크린샷 2022-02-16 오후 2 23 02" src="https://user-images.githubusercontent.com/68681500/154201467-fc8cd49d-0096-427a-868c-012ceee4235e.png">

해당 사진처럼 url로 백엔드 접속을 하게 된다면 

<img width="1435" alt="스크린샷 2022-02-16 오후 2 23 08" src="https://user-images.githubusercontent.com/68681500/154201483-821d5107-e111-482f-b377-d61bdf7957c9.png">

기존 로컬 백엔드 서버에서 서버가 실행되어있다는 문구가아닌 

축하 문구가 나오고 이동은 하지 않는데 어떻게 해결해야하나요..? AWS 사용이 처음이라 어느 부분이 문제인지 감이 안잡히네요..

**Comment by fsoftwareengineer** [comment]
우선 AWS를 처음으로 시도 해 보신 것을 축하드립니다!
제 생각에는 어플리케이션이 제대로 업로드 되지 않은 것 같습니다.

`.elasticbeanstalk/config.yml` 파일 전문을 코멘트에 남겨 주시면 확인 해 보도록 하겠습니다.

**Reply by Daehyeon2yam** [reply]
<img width="544" alt="스크린샷 2022-02-17 오전 11 39 14" src="https://user-images.githubusercontent.com/68681500/154394238-5e2f7c5c-f000-4999-a2dd-0c59573fc6bd.png">
안녕하세요 작가님 요청하신 config.yml 터미널에서 cat했을시의 화면입니다.

참고로 6.3.2의 init을 이용하여 환경을 초기화 하였으나, .ealsticbeanstalk의 데릭토리는 생성되지 않았지만
cat 명령어로 실행을 해보면 출력이 되어 vi명령어를 활용해 deploy를 추가해주었습니다.

**Reply by fsoftwareengineer** [reply]
감사합니다. config.yaml은 괜찮은 것 같습니다. eb deploy의 로그 화면도 추가 해주시면 확인 해 보겠습니다!

**Reply by Daehyeon2yam** [reply]
<img width="957" alt="스크린샷 2022-02-17 오후 3 16 25" src="https://user-images.githubusercontent.com/68681500/154416823-1e9f338d-300d-4ef6-9190-78177aaee26e.png">
경로의 문제라고 나와있는데 혹시 backend 애플리케이션 소스파일 위치에서 eb를 create했어야했나요??

**Reply by fsoftwareengineer** [reply]
책 기준으로 /demo 디렉터리 아래에서 eb create을 해 주셔야 합니다.

**Reply by doingH** [reply]
> 감사합니다. config.yaml은 괜찮은 것 같습니다. eb deploy의 로그 화면도 추가 해주시면 확인 해 보겠습니다!

.으로 시작하는 폴더와 파일은 숨김파일입니다.

**Comment by fsoftwareengineer** [comment, marked as answer]
책 기준으로 /demo 디렉터리 아래에서 eb create을 해 주셔야 합니다
